src/cr/cube/cubepart.py

```diff
@@ -916,6 +921,19 @@ def rows_scale_median_margin(self):
     def shape(self):
         return self.counts.shape
 
+    @lazyproperty
+    def share_sum(self):
+        """2D optional np.float64 ndarray of share of sum value for each table cell.
```

Comment:
I know that this is how we write docstrings for our api, but maybe you could add a paragraf, below everything, that describes "what the share sum is"... It would be nice to have for a reader of the api docs. Like, if I didn't know what it was, from your 1-1 explanation, I'd just stare at this 👓